clients/java/signalr/src/main/java/com/microsoft/aspnet/signalr/WebSocketTransport.java

```diff
@@ -72,31 +82,31 @@ public void stop() {
         logger.log(LogLevel.Information, "WebSocket connection stopped");
     }
 
-    private WebSocketClient createWebSocket() {
-        return new WebSocketClient(url) {
-             @Override
-             public void onOpen(ServerHandshake handshakedata) {
-                 System.out.println("Connected to " + url);
-             }
-
-             @Override
-             public void onMessage(String message) {
-                 try {
-                     onReceive(message);
-                 } catch (Exception e) {
-                     e.printStackTrace();
-                 }
-             }
-
-             @Override
-             public void onClose(int code, String reason, boolean remote) {
+    private WebSocketClient createWebSocket(Map<String, String> headers) {
+        return new WebSocketClient(this.url, headers) {
```

Comment:
Why are you now using `this` when accessing class members?

Reply:
I actually want to move towards using `this`. Since it's not the convention to use an underscore prefix I've started liking the idea of using `this`